client/src/components/common/NavBar.jsx

```diff
@@ -1,16 +1,21 @@
 import React from 'react';
+import { Navbar, Nav } from 'react-bootstrap';
 import { Link } from 'react-router-dom';
+import '/Users/tpl622_3/my-crochet-corner/client/src/App.css';
 
 const NavBar = () => {
   return (
-    <nav>
-      {/* creates links to the main pages */}
-        <h1>NavBar</h1>
-        <Link to="/home">Home</Link>
-        <Link to="/favorites">Favorites</Link>
-        <Link to="/profile">Profile</Link>
-    </nav>
-
+    <Navbar style={{ backgroundColor: '#FACDCC', padding: '10px', marginBottom: '20px' }} expand="lg">
```

Comment:
I tried making a className and Id for it to style in CSS, but I was unable to access it to style. I even tried using the React bootstrap name, and I am not sure why it will not let me access it, but it will let me access the others

Reply:
I tried using an Id that was not "navbar", and it worked! I think there may have been a confliction with it being a keyword? I'm not sure but I will try that for some of the other elements that I was not able to access through CSS!

Reply:
Submitted changes with commit: [7201e4f](https://github.com/abbiehuynh/my-crochet-corner/pull/13/commits/7201e4f09eb7ea2bf896af1129653364b02e160c)